# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1481719183 -3600
#      Wed Dec 14 13:39:43 2016 +0100
# Branch stable
# Node ID fd056e736f735d8a323baf6dd3f8d445d8e9d335
# Parent  e3fde9f1d47e9e3d7f48e89c76c7f43a80a545b0
languages/de_DE.ts: Fix spacing and punctuation (patch #9188)

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -2791,17 +2791,17 @@ oder wenn unterschiedliche Schriftgrößen oder -stile (fett/kursiv/normal) verwendet werden.</translation>
     <message>
         <location filename="../src/settings-dialog.cc" line="+127"/>
         <source>Octave Settings</source>
         <translation>Octave Einstellungen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save settings.  Missing settings file or unknown directory.</source>
-        <translation>Die Einstellungen können nicht gespeichert werden.  Fehlende Datei oder unbekanntes Verzeichnis.</translation>
+        <translation>Die Einstellungen können nicht gespeichert werden. Fehlende Datei oder unbekanntes Verzeichnis.</translation>
     </message>
     <message>
         <location line="+18"/>
         <location line="+4"/>
         <location line="+444"/>
         <source>System setting</source>
         <translation>Systemeinstellung</translation>
     </message>
@@ -2877,17 +2877,17 @@ a:link { text-decoration: underline; col
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;Allow Octave to connect to the Octave web site when it starts to display current news and information about the Octave community.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
-&lt;p&gt;Erlaube Octave beim Programmstart eine Verbindung zur Octave Webseite herzustellen um aktuelle Neuigkeiten und Informationen über Octave anzuzeigen.&lt;/p&gt;
+&lt;p&gt;Erlaube Octave beim Programmstart eine Verbindung zur Octave Webseite herzustellen, um aktuelle Neuigkeiten und Informationen über Octave anzuzeigen.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>shortcut_manager</name>
     <message>
         <location filename="../src/shortcut-manager.cc" line="+105"/>
         <source>New File</source>
